app.py

```diff
@@ -1,18 +1,69 @@
-"""main app file for modeler"""
+#"""main app file for modeler"""
```

Comment:
the leading `#` is not needed here, also you should add 2 blank lines between the doc level comment and the first code block.

if you want to use a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to indicate this is an executable file, it should look like this:
```
#!/usr/bin/env python
"""main app file for modeler"""
```

this will tell the shell to look for the python interpreter specified by the environment

Reply:
one more note, top level string objects are completely legal in python and the first encountered string will be used as the "docstring" for the module. more info in [pep 257](https://www.python.org/dev/peps/pep-0257/)